client/src/elm/Translate.elm

```diff
@@ -331,6 +331,9 @@ type TranslationId
     | AgeSingleMonthWithoutDay Int
     | AgeSingleDayWithMonth Int Int
     | AgeSingleDayWithoutMonth Int Int
+    | AgeSingleYearOld Int
+    | AgeSingleYearWithSingleMonth Int Int
+    | AgeSingleYearWithMonths Int Int
```

Comment:
Why we need first `Int`?
It's a single year, so it's translates to `One year and ...`.